db/ens.go

```diff
@@ -413,10 +413,14 @@ func (bigtable *Bigtable) ImportEnsUpdates(client *ethclient.Client) error {
 		if err := g.Wait(); err != nil {
 			return err
 		}
+		err = bigtable.WriteBulk(mutsDelete, bigtable.tableData)
+		if err != nil {
+			return err
+		}
 	}
 	logger.Info("ens key indexing completed")
 	// After processing the keys we remove them from bigtable
-	return bigtable.WriteBulk(mutsDelete, bigtable.tableData)
+	return err
```

Comment:
**Minor:** Should return `nil` to show that this path means everything's fine.
Also, the comment above now is misplaced, it should be moved to the `WriteBulk` code again.